ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -53,6 +79,32 @@ module Json = struct
   let merge_maps m maps =
     List.fold_left (fun acc map -> StringMap.union choose_enum acc map) m maps
 
+  let rec func_name_suffix = function
```

Comment:
In this case, could this be renamed to `suffix_of_type`? It could be used as `func_name_suffix` and `type_name_suffix`.

Reply:
Nice. It has been renamed to `suffix_of_type` in [5591](https://github.com/xapi-project/xen-api/pull/5591).